Dockerfile

```diff
@@ -38,6 +38,13 @@ RUN mkdir ./bin && \
 # runtime layer
 FROM alpine:3
 
+ENV arana_log_name=arana.log \
+    arana_log_level=0 \
+    arana_log_max_size=10 \
+    arana_log_max_backups=5 \
+    arana_log_max_age=30 \
+    arana_log_compress=false
```

Comment:
Use upper-case, for example, `ARANA_LOG_LEVEL`.

Reply:
done